cmd/run.go

```diff
@@ -435,19 +438,438 @@ func init() {
 	RootCmd.AddCommand(RunCmd)
 }
 
+func executeRun(runFilePath string, apps []runfileconfig.App) (bool, error) {
+	var exitWithError bool
+
+	// setup shutdown notify channel.
+	sigCh := make(chan os.Signal, 1)
+	setupShutdownNotify(sigCh)
+
+	runStates := make([]*runExec.RunExec, 0, len(apps))
+	for _, app := range apps {
```

Comment:
This starts each app in sequence, waiting until an app is ready before proceeding to the next?  If so, this could likely be perceived by users as being very slow.  We'll likely want to try to parallelize this.

Reply:
that is planned for the next version @philliphoff. Initial version is only sequential. But we might also need to add some `delay` between starting apps and a potential `dependsOn` when we parallelize the app start.

Reply:
Fair enough, and a `dependsOn` would be useful.